Tasks/HelmDeployV0/src/helm.ts

```diff
@@ -127,6 +127,7 @@ function runHelm(helmCli: helmcli, command: string, kubectlCli: kubernetescli, f
 
     const execResult = helmCli.execHelmCommand();
     tl.setVariable('helmExitCode', execResult.code.toString());
+    tl.setVariable('helmOutput', execResult.stdout.toString());
```

Comment:
execResult.stdout is already a string. You don't have to use .toString().

Reply:
Also, there should be a null check for execResult.stdout before setting `helmOutput`, because it might be null in case the command fails.

Reply:
Null check added and toString removed.